src/BarChart/BarChart.tsx

```diff
@@ -362,32 +349,19 @@ export class BarChart extends Component<BarChartProps> {
         )}
       </Fragment>
     );
-  }
-
-  render() {
-    const {
-      id,
-      width,
-      height,
-      margins,
-      className,
-      series,
-      xAxis,
-      yAxis
-    } = this.props;
+  };
 
-    return (
-      <ChartContainer
-        id={id}
-        width={width}
-        height={height}
-        margins={margins}
-        xAxisVisible={isAxisVisible(xAxis.props)}
-        yAxisVisible={isAxisVisible(yAxis.props)}
-        className={classNames(css.barChart, className, css[series.props.type])}
-      >
-        {(props) => this.renderChart(props)}
-      </ChartContainer>
-    );
-  }
-}
+  return (
+    <ChartContainer
+      id={id}
+      width={width}
+      height={height}
+      margins={margins}
+      xAxisVisible={isAxisVisible(xAxis.props)}
+      yAxisVisible={isAxisVisible(yAxis.props)}
+      className={classNames(css.barChart, className, css[series.props.type])}
+    >
+      {(props) => renderChart(props)}
```

Comment:
I think this could just be this:

```suggestion
      {renderChart}
```